pyuvsim/simsetup.py

```diff
@@ -1083,80 +1090,109 @@ def initialize_uvdata_from_params(obs_params):
     return uv_obj, beam_list, beam_dict
 
 
-def initialize_uvdata_from_keywords(yaml_filename=None, antenna_layout_filename=None, array_layout=None, telescope_location=None,
-                                    telescope_name=None, Nfreqs=None, start_freq=None, bandwidth=None, freq_array=None, channel_width=None,
-                                    Ntimes=None, integration_time=None, start_time=None, time_array=None,
-                                    bls=None, antenna_nums=None, antenna_names=None, polarizations=None, no_autos=False,
-                                    redundant_threshold=None, run_check=True, write_files=True, path_out=None, **kwargs):
+def initialize_uvdata_from_keywords(
+        yaml_filename=None, antenna_layout_filepath=None, output_layout_filename=None,
+        array_layout=None,
+        telescope_location=None, telescope_name=None, Nfreqs=None, start_freq=None,
+        bandwidth=None, freq_array=None, channel_width=None, Ntimes=None,
+        integration_time=None, start_time=None, time_array=None, bls=None,
+        antenna_nums=None, antenna_names=None, polarization_array=None, no_autos=False,
+        redundant_threshold=None, write_files=True, path_out=None, complete=False,
+        **kwargs):
     """
     Setup a UVData object from keywords.
```

Comment:
This description could probably be a bit more informative, saying something about how it writes a yaml file.

Reply:
I've added an explanation that it will optionally write configs that can be used by `initialize_uvdata_from_params`.